docs/install/microsoft-azure-aks.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Microsoft Azure Kubernetes Service (AKS)"
+linkTitle: "Microsoft Azure Kubernetes Service (AKS)"
```

Comment:
This is the title that will be rendered in the side menu. I believe it will be very long for that column and may be break.

Reply:
Good point - should we remove "Microsoft"? I did see on the "Docker Enterprise Edition" that it auto wraps the title in the side menu if you make your window smaller. 

@mdeneva what do you think?